circus-train-core/src/test/resources/log4j.xml

```diff
@@ -25,11 +25,14 @@
 
   <logger name="com.hotels.bdp.circustrain">
     <!-- Required to be DEBUG for com.hotels.bdp.circustrain.core.metastore.HiveConfFactoryTest.secretProperties() -->
```

Comment:
remove the comment please. Setting the log level in the test is a better idea

Reply:
ok